tests/foreman/api/test_content_credentials.py

```diff
@@ -0,0 +1,233 @@
+"""Unit tests for the ``gpgkeys`` paths.
+
+:Requirement: ContentCredential
+
+:CaseAutomation: Automated
+
+:CaseLevel: Component
+
+:CaseComponent: ContentCredentials
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+import pytest
+from fauxfactory import gen_string
+from nailgun import entities
+from requests import HTTPError
+
+from robottelo.constants import VALID_GPG_KEY_BETA_FILE
+from robottelo.constants import VALID_GPG_KEY_FILE
+from robottelo.datafactory import invalid_values_list
+from robottelo.datafactory import parametrized
+from robottelo.datafactory import valid_data_list
+from robottelo.helpers import read_data_file
+
+
+key_content = read_data_file(VALID_GPG_KEY_FILE)
+
+
+@pytest.mark.tier1
+def test_positive_search_in_org(module_org):
+    """Search for a GPG key and specify just ``organization_id``.
+
+    :id: ff5e047c-404b-4379-8d28-3ad8cb39b6a9
+
+    :Steps:
+
+        1. Create an organization.
+        2. Create a GPG key belonging to the organization.
+        3. Search for GPG keys in the organization.
+
+    :expectedresults: Only one GPG key is in the search results: the
+        created GPG key.
+
+    :CaseImportance: Critical
+    """
+    org = entities.Organization().create()
```

Comment:
Why are you creating an organization again?

Reply:
hmm, I missed that. I need to correct the doc string too.